server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/tservers.ftl

```diff
@@ -25,9 +25,9 @@
       </div>
       <div class="row">
         <div class="col-xs-12">
-          <table id="badtservers" class="table table-bordered table-striped table-condensed" style="display: none;">
```

Comment:
So how are you hiding the table now? I didn't see a hide anywhere. I think with the other datatables I had to hide the div around the table.